apps/store/themes/store/js/list_assets.js

```diff
@@ -490,114 +490,116 @@ function resolveDomain() {
 }
 
 $(window).load(function () {
+    if (store.taxonomyAvailability) {
 // this condition will check current page is asset list page or top asset page.
-    if (window.location.href.toString().indexOf('list') > 0 || window.location.href.toString().
-            indexOf('top-assets') > 0) {
-
-        // first ajax call when page loads
-        $.ajax({
-            url: caramel.context + '/apis/taxonomies?' + resolveDomain(),
-            type: 'GET',
-            async: false,
-            headers: {
-                Accept: "application/json"
-            },
-            success: function (data) {
-                // success function will modify the REST api return data into jstree format.
-                var children;
-                try {
-                    children = Array.isArray(data[0].children);
-                } catch (e) {
+        if (window.location.href.toString().indexOf('list') > 0 || window.location.href.toString().
```

Comment:
Thank you Sameera . This entire code block has been refactored with multiple taxonomy. Its not available anymore.